packages/ui/src/Table.tsx

```diff
@@ -311,16 +322,15 @@ export const Column = styled.td<{
   width?: number | string
   alignText?: string
   withTitle?: boolean
-  isTitleAString?: boolean
 }>`
   border-top: 1px solid ${({ theme }) => theme.tableBorderColor};
   background-color: ${({ theme }) => theme.tableCellBackgroundColor};
   color: ${({ theme }) => theme.tableTextColor};
   font-family: ${Fonts.Inter}, serif;
   font-weight: 600;
   font-size: 16px;
-  ${({ width, alignText, withTitle, isTitleAString }) => `
-      padding: ${!isTitleAString && withTitle ? '15px' : '30px'} 20px;
+  ${({ width, alignText }) => `
+      padding: 15px 20px;
       text-align: ${alignText || 'center'};
 
       ${
```

Comment:
@aleksaToljic here we should probably calculate width as **width in percentage - left & right padding**
e.g. **calc(50% - 40px)** . But I couldn't achieve this, calc not working.